godel_plugins/include/godel_plugins/widgets/robot_blending_widget.h

```diff
@@ -55,93 +57,12 @@ const std::string SURFACE_BLENDING_PARAMETERS_SERVICE = "surface_blending_parame
 const std::string SELECT_SURFACE_SERVICE = "select_surface";
 const std::string PROCESS_PATH_SERVICE="process_path";
 const std::string SELECTED_SURFACES_CHANGED_TOPIC = "selected_surfaces_changed";
+const std::string GET_AVAILABLE_MOTION_PLANS_SERVICE = "get_available_motion_plans";
+const std::string SELECT_MOTION_PLAN_SERVICE = "select_motion_plan";
+const std::string LOAD_SAVE_MOTION_PLAN_SERVICE = "load_save_motion_plan";
+const std::string RENAME_SURFACE_SERVICE = "rename_surface";
 
 
-class PoseWidget: public QWidget
-{
-Q_OBJECT
-public:
-	PoseWidget(QWidget *parent = NULL);
-
-	void set_values(const geometry_msgs::Pose& p);
-	void set_values(const tf::Transform &t);
-	tf::Transform get_values();
-
-protected:
-
-	Ui::PoseWidget ui_;
-};
-
-class RobotScanConfigWidget: public QMainWindow
-{
-
-private:
-
-Q_OBJECT
-public:
-
-	RobotScanConfigWidget(godel_msgs::RobotScanParameters params);
-	void show();
-
-Q_SIGNALS:
-	void parameters_changed();
-
-protected:
-
-	void init();
-	void update_parameters();
-	void save_parameters();
-
-protected Q_SLOTS:
-
-	void accept_changes_handler();
-	void cancel_changes_handler();
-
-public:
-
-	godel_msgs::RobotScanParameters robot_scan_parameters_;
-
-protected:
-
-	Ui::RobotScanConfigWindow ui_;
-	PoseWidget *world_to_obj_pose_widget_;
-	PoseWidget *tcp_to_cam_pose_widget_;
-};
-
-class SurfaceDetectionConfigWidget: public QMainWindow
-{
-
-private:
-
-Q_OBJECT
-public:
-
-	SurfaceDetectionConfigWidget(godel_msgs::SurfaceDetectionParameters params);
-	void show();
-
-Q_SIGNALS:
-	void parameters_changed();
-
-protected:
-
-	void init();
-	void update_parameters();
-	void save_parameters();
-
-protected Q_SLOTS:
-
-	void accept_changes_handler();
-	void cancel_changes_handler();
-
-public:
-
-	godel_msgs::SurfaceDetectionParameters surface_detection_parameters_;
-
-protected:
-
-	Ui::SurfaceDetectionConfigWindow ui_;
-};
-
 class RobotBlendingWidget:  public QWidget
```

Comment:
Out of curiosity, is the RobotBlendingWidget class taking on the role of the deleted Widged classes in this file?